legacy/application/models/Preference.php

```diff
@@ -1399,6 +1399,16 @@ public static function setRadioPageDisplayLoginButton($value)
         self::setValue('radio_page_display_login_button', $value);
     }
 
+    public static function getSmartblockTrimOverbooked()
+    {
+        return self::getValue('smartblock_trim_overbooked');
```

Comment:
Could we rename this key to `schedule_trim_overbooked` (and all related function/fields names), we are effectively touching the schedule table.

Reply:
Also, we might want to add a default either in the code, or in the database (using the default DB data + migrations).

In the code seem easier/cleaner.

Reply:
rename sounds logical. there is no verbatim default cause the default is "false" which is equal to an unset database field or empty string. This is actually checked in the preferences dalogue after the getter is used. So all installations will default to 'false' which is the original behaviour.